src/components/LoanCards/LoanCardImage.vue

```diff
@@ -4,6 +4,7 @@
 			:to="`/lend/${loanId}`"
 			v-kv-track-event="['Lending', 'click-Read more', 'Photo', loanId, 'true']"
 			class="borrower-image-link"
+			:target="linkTarget"
```

Comment:
@emuvente for these loans shown on the basket we don't want to close that page so we open a new tab. An interesting feature for "router-link" but hey...guess we weren't the only ones that needed it.